Editor/Modules/FogModeAnalyzer.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Unity.ProjectAuditor.Editor.Core;
+using Unity.ProjectAuditor.Editor.Diagnostic;
+using Unity.ProjectAuditor.Editor.Modules;
+using UnityEngine;
+
+namespace Unity.ProjectAuditor.Editor.Modules
+{
+    class FogModeAnalyzer : ISettingsModuleAnalyzer
+    {
+        private static readonly Descriptor k_fogModeDescriptor = new Descriptor(
```

Comment:
Code style note:  The symbol should be "k_FogModeDescriptor" (capital "F") it seems, comparing to other code.

So a capital letter after the "k_" prefix for any kind of constant fields basically.

Reply:
(line removed)